Packs/Cybereason/ReleaseNotes/2_1_17.md

```diff
@@ -0,0 +1,13 @@
+
+#### Integrations
+
+##### Cybereason
+
+- Added a new command ***cybereason-process-attack-tree*** to fetch Process attack tree url.
```

Comment:
Replace "Added a new command ***cybereason-process-attack-tree*** to fetch Process attack tree url." with
"Added a new command ***cybereason-process-attack-tree*** to return Attack tree url for a given Process."

Reply:
Replaced